### ISSUE TITLE
Not all fields working

### ISSUE THREAD

**Issue by knobbyloafer** [tomdotorg/weather#38]
The common fields listed work as expected when uploading but some outside the sample set do not.  For example UV and solarradiation.

**Comment by tomdotorg** [comment]
Good point. I have only used and tested the fields I have sensors for. But I do use solar radiation and I thought it was OK. UV I get - never touched. it.

Feel free to do a PR for those things - I'd love to include them!

**Comment by knobbyloafer** [comment]
Thanks for the reply.  It appears to be working.  When I tried again with the same code, the values updated.  UV and Solar do not dynamically update (while all others do) on the weatherunderground page so that may have been the issue.  I could have sworn I tried refreshing though.  Anyway, it looks like a false alarm.

**Comment by tommitchell-eis** [comment]
Great. I'd love to see your site - can you share it?
